packages/backend/src/App.ts

```diff
@@ -494,6 +494,16 @@ export default class App {
             next();
         });
 
+        expressApp.use((req, res, next) => {
+            if (req.user) {
+                Sentry.setUser({
+                    id: req.user.userUuid,
+                    organization: req.user.organizationUuid,
+                });
+            }
+            next();
+        });
```

Comment:
> The requestDataIntegration will no longer automatically set the user from request.user when express is used. Starting in v9, you'll need to manually call Sentry.setUser() e.g. in a middleware to set the user on Sentry events.

Reply:
can you update the `SchedulerTaskTracer` `setUser` to include 'organization': 'payloadTags['organization.uuid']'?